pages/create.js

```diff
@@ -0,0 +1,33 @@
+import PlantForm from "@/components/PlantForm";
+import useSWR from "swr";
+import { useRouter } from "next/router";
+import Router from "next/router";
+
+export default function CreatePage() {
+  const { data: plants, error, isLoading, mutate } = useSWR("/api/plants");
+  const router = useRouter();
+  async function handleAddPlant(newPlantData) {
+    const response = await fetch("/api/plants", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(newPlantData),
+    });
+
+    if (response.ok) {
+      Router.push("/plants");
```

Comment:
You’re mixing `Router.push` (the singleton router, the global instance) and `router.push` (the `useRouter` hook instance).

While both work, `router.push` from `useRouter` is the preferred approach for consistency within functional components.

```suggestion
      router.push("/plants");
```